packages/ui-extensions/src/surfaces/customer-account/api/docs.ts

```diff
@@ -77,3 +78,48 @@ export interface Docs_Standard_QueryApi
 export interface Docs_StandardApi extends Omit<StandardApi<any>, 'router'> {}
 
 export interface Docs_FullPageApi extends Omit<FullPageApi, 'location'> {}
+
+export interface Docs_Page_Button_PrimaryAction
+  extends Pick<
+    ButtonProps,
+    | 'onPress'
+    | 'to'
+    | 'loading'
+    | 'loadingLabel'
+    | 'disabled'
+    | 'accessibilityLabel'
+  > {}
+export interface Docs_Page_Button_SecondaryAction
+  extends Pick<
+    ButtonProps,
+    'onPress' | 'to' | 'loading' | 'loadingLabel' | 'disabled'
+  > {
+  /**
+   * A label used for buyers using assistive technologies. Needed because `children` passed to this component will be discarded.
+   */
+  accessibilityLabel: ButtonProps['accessibilityLabel'];
+}
+
+export interface Docs_ResourceItem_Button_Action
+  extends Pick<
+    ButtonProps,
+    | 'onPress'
+    | 'to'
+    | 'loading'
+    | 'loadingLabel'
+    | 'disabled'
+    | 'accessibilityLabel'
+  > {}
+
+export interface Docs_OrderActionMenu_Button
+  extends Pick<
+    ButtonProps,
+    'onPress' | 'loading' | 'loadingLabel' | 'disabled' | 'accessibilityLabel'
```

Comment:
Do we accept all of this? I think we only picked the
`text` `to` 
https://github.com/Shopify/customer-account-web/blob/main/app/foundation/extensibility/extensions/menu-items/useMenuItems.tsx#L59-L67

Reply:
I think we will, but I could remove it until we actually do and we'd add it once it's there for real.

Reply:
I'll check how much effort it is to add it to the host. I'm ok, if this PR here is open for a day or so

Reply:
It's not a lie anymore :D